tests/unit/test_messagepack.py

```diff
@@ -167,9 +167,8 @@ def _required_arg_test_ids(param):
         (EPStatusReport, {"endpoint_id": ID_ZERO, "ep_status_report": {}}),
         # ManagerStatusReport requires: task_statuses
         (ManagerStatusReport, {}),
-        # Task requires: task_id, container_id, task_buffer
+        # Task requires: task_id, task_buffer
         (Task, {"container_id": ID_ZERO, "task_buffer": "foo data"}),
-        (Task, {"task_id": ID_ZERO, "task_buffer": "foo data"}),
```

Comment:
Would you be alright with making the change to add this to the positive test-cases above? (`test_pack_and_unpack`)

I know testing it may seem very picky, but I want the tests to cover all of the kinds of message which we pass through tis layer. If `container_id = None` is a case we want to support, it's best to include it in the testing.

Reply:
Good call, should probably have looked out for that. Pushed